test/secret/docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020-2021 - Offen Authors <hioffen@posteo.de>
+# SPDX-License-Identifier: Unlicense
+
+version: '3.8'
+
+services:
+  minio:
```

Comment:
The tests are failing right now. All that is needed to make them pass should be creating the target bucket in the `minio` service's entrypoint like in the `swarm` test:

https://github.com/offen/docker-volume-backup/blob/00c83dfac79af6f03c677e187b5bce6817b2c2a7/test/swarm/docker-compose.yml#L7-L19

(this is very non-obivous)

Reply:
I have made a different approach, where a sidecart is running first to create the directory. ( Troublesome, but imo its a tad bit better than entrypoint+command magic )

There was/is also some issue with minio versions, the latest one seems to store the files encoded(?), so the test couldn't just access the tar natively. I found some older version, that already supports the _FILE variables, but doesn't have this problem. Not a future proof solution, but that's all I've had time for now.